api/src/main/java/org/pmiops/workbench/db/WorkbenchDbConfig.java

```diff
@@ -36,14 +37,28 @@ public class WorkbenchDbConfig {
 
   @Primary
   @Bean(name = "dataSourceProperties")
-  @ConfigurationProperties(prefix = "spring.datasource")
   public DataSourceProperties dataSourceProperties() {
-    return new DataSourceProperties();
+    DataSourceProperties p = new DataSourceProperties();
+    // DB_HOST should be defined for local development, otherwise CLOUD_SQL_INSTANCE_NAME is
+    // required.
+    Optional<String> dbHost = getEnv("DB_HOST");
```

Comment:
The right way to make this work for tests is to use a class here that can be replaced via dependency injection in tests.